tests/src/python/test_qgsimagecache.py

```diff
@@ -158,6 +158,37 @@ def testRemoteImageBlocking(self):
             )
         )
 
+    def testInvalidateCacheEntry(self):
+        """Test invalidating a cache entry."""
+        cache = QgsApplication.imageCache()
+
+        temp_image_path = os.path.join(
+            os.getenv("TMPDIR", "/tmp"), "test_sample_image.png"
+        )
+        original_image = os.path.join(TEST_DATA_DIR, "sample_image.png")
+        if os.path.exists(temp_image_path):
+            os.remove(temp_image_path)
+        self.assertTrue(os.path.exists(original_image))
+        self.assertTrue(os.path.isfile(original_image))
+        os.system(f"cp {original_image} {temp_image_path}")
```

Comment:
This will not work on Windows, will it? Maybe it would be better to replace it with the method from `shutil` module?

Reply:
Thanks @alexbruy, this is fixed now (and rebased/force pushed)